pypicloud/storage/s3.py

```diff
@@ -48,6 +56,17 @@ def configure(cls, settings):
                                  'aws.access_key', None)
         secret_key = getdefaults(settings, 'storage.secret_key',
                                  'aws.secret_key', None)
+        host = getdefaults(settings, 'storage.host',
+                           'aws.host', boto.s3.connection.NoHostProvided)
+        is_secure = getdefaults(settings, 'storage.is_secure',
+                                'aws.is_secure', True)
+        calling_format = getdefaults(settings, 'storage.calling_format',
+                                     'aws.calling_format',
+                                     'SubdomainCallingFormat')
```

Comment:
You can just use the standard `settings.get('storage.calling_format', 'SubdomainCallingFormat')` syntax for these options. The only reason the others are using `getdefaults()` is to maintain backwards compatibility with old config options. You'll notice that `aws.access_key` and the others aren't even listed in the documentation anymore.